lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -125,7 +128,7 @@ export class EvaluationClientWrapper {
       const idRes = JsonPathUtils.extractInputField(pd, [idPath]);
       if (idRes.length) {
         const submissionRequirementMatch: SubmissionRequirementMatch = {
-          name: idRes[0].value.name,
```

Comment:
What is the reason we are removing the name?

Reply:
It was a mistake